docs/go/how-to-set-tls-configuration-in-go.md

```diff
@@ -0,0 +1,26 @@
+---
+id: how-to-set-tls-configuration-in-go
+title: How to set TLS configuration in Go
+sidebar_label: Set TLS configuration
+description: Set TLS configuration
+tags:
+  - developer-guide
+  - sdk
+  - Go
+---
+
+Use [`ConnectionOptions`](https://pkg.go.dev/go.temporal.io/sdk@v1.15.0/client#ConnectionOptions) API available in the [`go.temporal.io/sdk/client`](https://pkg.go.dev/go.temporal.io/sdk/client) package to connect a Client with TLS.
```

Comment:
Do we want to link to a specific version?
Do we make sure to update all the links when a new version is released?

Reply:
I see, it's just that you linked to a specific version here, so I asked.